swarm/network/protocol_test.go

```diff
@@ -30,6 +30,8 @@ import (
 	p2ptest "github.com/ethereum/go-ethereum/p2p/testing"
 )
 
+const DefaultVersion = 5
```

Comment:
This should be `6`, no?

Reply:
I changed it to TestProtocolNetworkID. As far as I understand it can be anything that is not 0 based on the existing tests. Feel free to correct me if I am wrong.